### ISSUE TITLE
Publish all source files in the transparency portal

### ISSUE THREAD

**Issue by akariv** [os-data/mexican-federal-budget#38]
For us to be able to develop the 'auto-updating' pipeline (as well as for the sake of open data and transparency), please publish all source files:
- CSV files, one per year
- Catalog files as needed
- Also consider the column aliases file

In a public, official website - preferably your open data portal

**Comment by Victordeleon** [comment]
Hey @akariv sorry to bother, but I had to replace the URL for 2016, since it wouldn't be permanent (I would have to give you a new one for the third and the fourth semester). So I replaced the name of the file and of the link, so I can substite future 2016 actualizations without changing the URL again.

The rest of the years remain the same.

https://docs.google.com/spreadsheets/d/105EkG0R5amAN0RnqLH6ew_qYYqtlsGcaVFI5BwzHMLs/edit#gid=283354147
